pos_customer_wallet/static/src/js/screens.js

```diff
@@ -49,6 +54,15 @@ odoo.define("pos_customer_wallet.screens", function (require) {
                     });
                     return false;
                 }
+                if (product_lines_qty > 0) {
+                    this.gui.show_popup("error", {
+                        title: _t("No customer selected"),
+                        body: _t(
+                            "Cannot use customer wallet Product without selecting a customer.\n\n Please select a customer or use a different payment method."
```

Comment:
Can we add the name of the product after 'customer wallet Product'?

Reply:
Indeed. Done.


(mark as solved)